lib/elixir/lib/calendar.ex

```diff
@@ -151,11 +244,18 @@ defmodule Date do
       true
 
   """
-  @spec utc_today() :: t
-  def utc_today() do
+  @spec utc_today(Calendar.calendar) :: t
+  def utc_today(calendar \\ Calendar.ISO)
+  def utc_today(Calendar.ISO) do
     {:ok, {year, month, day}, _, _} = Calendar.ISO.from_unix(:os.system_time, :native)
     %Date{year: year, month: month, day: day}
   end
+  def utc_today(calendar) do
+    {:ok, {year, month, day}, _, _} = Calendar.ISO.from_unix(:os.system_time, :native)
+    %Date{year: year, month: month, day: day, calendar: Calendar.ISO}
```

Comment:
Hrm, why are you not using convert on this case? I think this is the only `utc_today` implementation that does not require inlining (as long as we use convert which is already optimized).